docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -12,11 +12,14 @@ When a process instance arrives at such a user task, a new job is created. The p
 
 While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial.
 
-In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser.
+In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser. If you rather prefer a video based learning experience we got you covered as well. [This Camunda Academy course](https://bit.ly/3PJJocB) features the same content which is being discussed in this guide.
 
 :::note
-New to BPMN or want to learn more? Visit our [BPMN cheat sheet](https://page.camunda.com/wp-bpmn-2-0-business-process-model-and-notation-en) for an overview of all BPMN symbols.
-:::
+Getting stuck? Book a meeting with our User Success Specialists. Follow the links dependent on the location you are joining:
+
+- [American regions](https://meetings.salesloft.com/camunda/jonathanjohnson)
```

Comment:
Same comment here as above.

Reply:
Yes, this link is valid for the whole American continent. :)